h2o-admissibleml/src/main/java/hex/schemas/InfogramV3.java

```diff
@@ -0,0 +1,246 @@
+    public long seed;
+
+    // Input fields
+    @API(help = "Standardize numeric columns to have zero mean and unit variance", level = API.Level.critical)
+    public boolean standardize;
+
+    @API(help = "Plug Values (a single row frame containing values that will be used to impute missing values of the" +
+            " training/validation frame, use with conjunction missing_values_handling = PlugValues)", 
+            direction = API.Direction.INPUT)
+    public KeyV3.FrameKeyV3 plug_values;
+
+/*    @API(help = "Restrict coefficients (not intercept) to be non-negative")
+    public boolean non_negative;*/
+
+    @API(help = "Maximum number of iterations", level = API.Level.secondary)
+    public int max_iterations;
+
+    @API(help = "Prior probability for y==1. To be used only for logistic regression iff the data has been sampled " +
+            "and the mean of response does not reflect reality.", level = API.Level.expert)
+    public double prior;
+
+    // dead unused args, formely inherited from supervised model schema
+    /**
+     * For imbalanced data, balance training data class counts via
+     * over/under-sampling. This can result in improved predictive accuracy.
+     */
+    @API(help = "Balance training data class counts via over/under-sampling (for imbalanced data).", 
+            level = API.Level.secondary, direction = API.Direction.INOUT)
+    public boolean balance_classes;
+
+    /**
+     * Desired over/under-sampling ratios per class (lexicographic order).
+     * Only when balance_classes is enabled.
+     * If not specified, they will be automatically computed to obtain class balance during training.
+     */
+    @API(help = "Desired over/under-sampling ratios per class (in lexicographic order). If not specified, sampling" +
+            " factors will be automatically computed to obtain class balance during training. Requires " +
+            "balance_classes.", level = API.Level.expert, direction = API.Direction.INOUT)
+    public float[] class_sampling_factors;
+
+    /**
+     * When classes are balanced, limit the resulting dataset size to the
+     * specified multiple of the original dataset size.
+     */
+    @API(help = "Maximum relative size of the training data after balancing class counts (can be less than 1.0). " +
+            "Requires balance_classes.", /* dmin=1e-3, */ level = API.Level.expert, direction = API.Direction.INOUT)
+    public float max_after_balance_size;
+
+    /** For classification models, the maximum size (in terms of classes) of
+     *  the confusion matrix for it to be printed. This option is meant to
+     *  avoid printing extremely large confusion matrices.  */
+    @API(help = "[Deprecated] Maximum size (# classes) for confusion matrices to be printed in the Logs", 
+            level = API.Level.secondary, direction = API.Direction.INOUT)
+    public int max_confusion_matrix_size;
```

Comment:
The deprecated comment was added by angela.  I don't know why she did that because we do need to compute the confusion matrix.  I removed her deprecated comment because I know michalk will work on that at some point.